javascript/javascript3/week3/promisesRepo/index.js

```diff
@@ -0,0 +1,48 @@
+//Getting into promises
+
+const repo = ['simonreddy2001', 'greeshmarahool', 'sarulathaanbu'];
+const body = document.querySelector('body');
+
+async function getRepo(repo){
+    const response = await fetch(`https://api.github.com/search/repositories?q=user:${repo}`);
+    const data = await response.json();
+    console.log(data); 
+    const h2 = document.createElement('h2');
+    h2.innerHTML = `${repo} Repository  `;
+    body.appendChild(h2);
+    const ul = document.createElement('ul');
+    h2.appendChild(ul);
+    data.items.map((item) => {
+       /*  console.log(`Name: ${item.name}`);
+        console.log(`URL: ${item.html_url}`);
+        console.log(`Owner: ${item.owner.login}`); */
+    const li = document.createElement('li');
```

Comment:
Please use more descriptive variable names eg. `repositoryList` instead of `ul` and `repositoryItem` instead of `li`

Reply:
Yes, I have changed them according to your suggestion:)